src/inference/src/threading/ie_parallel_custom_arena.cpp

```diff
@@ -12,7 +12,7 @@
 
 #    define USE_TBBBIND_2_5                 (TBBBIND_2_5_AVAILABLE && TBB_INTERFACE_VERSION < 12020)
 #    define TBB_NUMA_SUPPORT_PRESENT        (TBB_INTERFACE_VERSION >= 11100)
-#    define TBB_HYBRID_CPUS_SUPPORT_PRESENT (TBB_INTERFACE_VERSION >= 12020)
```

Comment:
let's add such condition only for apple. And we need to compare `12020` with actual number from 2021.3 TBB release

Reply:
Fixed